src/atoms/shortenText/shortenText.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { SimpleTooltip } from '../../molecules/tooltip/TooltipComponent';
+import { Text } from '../texts/text';
+import { ShortenTextProps } from './types';
+
+export const ShortenText = ({
```

Comment:
I might be a bit picky here but maybe we can just call it ShortText?

Reply:
haha, yeah a bit picky, but agreed and changed